crontab.sh

```diff
@@ -21,8 +21,10 @@
 30 0 * * 1 /bin/bash schedule/jobs/us-covid-forecast.sh > schedule/logs/us-covid-forecast.log 2>&1
 # run the Germany/Poland forecast on Monday at 4 am
 0 4 * * 1 /bin/bash schedule/jobs/germany-covid-forecast.sh > schedule/logs/germany-covid-forecast.log 2>&1
-# update crowd forecasts on a Tuesday Evening at 7pm
-0 19 * * 2 /bin/bash schedule/jobs/germany-crowd-forecast.sh > schedule/logs/germany-crowd-forecast.log 2>&1
+# update crowd forecasts on Saturday at 5 pm
+0 17 * * 6 /bin/bash schedule/jobs/germany-crowd-forecast-data-update.sh > schedule/logs/germany-crowd-forecast-data-update.log 2>&1
```

Comment:
I'm just wondering if the German forecasting jobs should live in a subfolder 

so `schedule/jobs/germany-covid-forecast/crowd-data-update.sh` etc or otherwise.